toonz/sources/image/sgi/filesgi.cpp

```diff
@@ -1232,7 +1232,7 @@ void Tiio::SgiWriterProperties::updateTranslation() {
   m_pixelSize.setItemUIName(L"48 bits", tr("48 bits"));
   m_pixelSize.setItemUIName(L"64 bits", tr("64 bits"));
   m_pixelSize.setItemUIName(L"8 bits (Greyscale)", tr("8 bits (Greyscale)"));
-  m_endianess.setQStringName(tr("Endianess"));
+  m_endianess.setQStringName(tr("Endianness"));
```

Comment:
This line is OK to be fixed.
The function `tr()` is for registering a word to the translation source file (.ts file). It is usually used for the terms in user interface like labels, command names and messages.
Fixing the typo of the words in the `tr()` function would be much appreciated as they are visible to users 👍